src/WebApiContrib.Core.Formatter.Protobuf/ProtobufFormatterOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿
+using System.Collections.Generic;
+
+namespace WebApiContrib.Core.Formatter.Protobuf
+{
+    public class ProtobufFormatterOptions
+    {
+        //public IFormatterResolver FormatterResolver { get; set; } = ContractlessStandardResolver.Instance;
+
+        public HashSet<string> SupportedContentTypes { get; set; } = new HashSet<string> { "application/x-protobuf", "application/x-protobuf" };

```

Comment:
Think so as well, be another client uses this

From SO


  	flag
	
looks like protorpc switched to application/octet-stream: github.com/google/protorpc/commit/… – Marc Gravell♦ May 28 '15 at 12:01